common.go

```diff
@@ -87,6 +88,12 @@ func GetLanguageByClassifier(content []byte, candidates []string) (language stri
 	return getLanguageByStrategy(GetLanguagesByClassifier, "", content, candidates)
 }
 
+// GetLanguageByGitAttributes returns the language assigned for a given regular expresion in .gitattributes.
+// This strategy needs to be initialized callin LoadGitAttributes
```

Comment:
What is the argument to this function? a filename or a regular expression (function signature says the former, comment explains the later)